Calibration/HcalConnectivity/src/HcalCableMapper.cc

```diff
@@ -159,17 +158,13 @@ void HcalCableMapper::process(const PathSet &ps, const IdMap &im) {
 }
 
 void HcalCableMapper::analyze(edm::Event const &e, edm::EventSetup const &c) {
-  edm::Handle<HBHEDigiCollection> hbhe;
-  e.getByToken(tok_hbhe_, hbhe);
-
-  edm::Handle<HFDigiCollection> hf;
-  e.getByToken(tok_hf_, hf);
-  edm::Handle<HODigiCollection> ho;
-  e.getByToken(tok_ho_, ho);
+  const edm::Handle<HBHEDigiCollection> &hbhe = e.getHandle(tok_hbhe_);
+  const edm::Handle<HFDigiCollection> &hf = e.getHandle(tok_hf_);
+  const edm::Handle<HODigiCollection> &ho = e.getHandle(tok_ho_);
 
-  record(*hbhe);
-  record(*hf);
-  record(*ho);
+  record(*(hbhe.product()));
+  record(*(hf.product()));
+  record(*(ho.product()));
 }
```

Comment:
The handle is not used for anything (no check in the validity) so one could just directly get the product from the event instead of the handle.